modules/post/windows/gather/credentials/spark_im.rb

```diff
@@ -134,7 +138,7 @@ def run
         print_status("#{file} does not contain any saved passwords")
         # close file and return
         config.close
-        return
+        next
       end
```

Comment:
I think `password.nil?` can never be true here

```
> contents = ''
> contents.split("\n").grep(/password/)
 => []
```

Reply:
Changed to `blank?`.